SPECS/xenopsd.spec

```diff
@@ -95,20 +114,54 @@ cp %{SOURCE6} xenopsd-network-conf
 cp %{SOURCE7} xenopsd-64-conf
 
 %build
+# this is a hack: we build and install two builds into the source
+# directory under root1 and root2. In the install step all we do is
+# copying files from it.
+
+mkdir root1 root2
 ./configure --libexecdir %{_libexecdir}/%{name}
+
+# regular build
+make
+make install DESTDIR=$PWD/root1 LIBEXECDIR=%{_libexecdir}/%{name} SBINDIR=%{_sbindir} MANDIR=%{_mandir} 
+make clean
+
+# now build for coverage profiling
+make coverage
 make
+make install DESTDIR=$PWD/root2 LIBEXECDIR=%{_libexecdir}/%{name} SBINDIR=%{_sbindir} MANDIR=%{_mandir} 
 
 %install
-make install DESTDIR=%{buildroot} LIBEXECDIR=%{_libexecdir}/%{name} SBINDIR=%{_sbindir} MANDIR=%{_mandir} 
+# this installs the files from the first build
+rsync -a root1/ %{buildroot} 
+
+# rename regular binaries
+mv %{buildroot}%{_sbindir}/xenopsd-xc                     %{buildroot}%{_sbindir}/xenopsd-xc.bin
+mv %{buildroot}%{_libexecdir}/%{name}/set-domain-uuid     %{buildroot}%{_libexecdir}/%{name}/set-domain-uuid.bin
+# mv %{buildroot}%{_sbindir}/xenopsd-xenlight       %{buildroot}%{_sbindir}/xenopsd-xenlight.bin 
+# mv %{buildroot}%{_sbindir}/xenopsd-simulator      %{buildroot}%{_sbindir}/xenopsd-simulator.bin
+
+# install selected binaries with coverage profiling from second build
+install -D ./xenops_xc_main.native        %{buildroot}%{_sbindir}/xenopsd-xc.cov
+install -D ./set_domain_uuid.native       %{buildroot}%{_libexecdir}/%{name}/set-domain-uuid.cov
+# install -D ./xenops_xl_main.native        %{buildroot}%{_sbindir}/xenopsd-xenlight.cov
+# install -D ./xenops_simulator_main.native %{buildroot}%{_sbindir}/xenopsd-simulator.cov
+
+touch %{buildroot}%{_sbindir}/xenopsd-xenlight
```

Comment:
Please add a short comment explaining why these files are touched.   (It's to do with the %ghost lines below).